src/bidiMapper/CommandProcessor.ts

```diff
@@ -266,8 +266,8 @@ export class CommandProcessor extends EventEmitter<CommandProcessorEventsMap> {
       // Emulation module
       // keep-sorted start block=yes
       case 'emulation.setGeolocationOverride':
-        throw new UnknownErrorException(
-          `Method ${command.method} is not implemented.`,
+        return await this.#browsingContextProcessor.setGeolocationOverride(
```

Comment:
From my understanding we wanted to have a processor per module?
This breaks this assumption.

Reply:
good point. Will do.

Reply:
done